format/spec.md

```diff
@@ -632,6 +632,29 @@ When expiring snapshots, retention policies in table and snapshot references are
     2. The snapshot is not one of the first `min-snapshots-to-keep` in the branch (including the branch's referenced snapshot)
 5. Expire any snapshot not in the set of snapshots to retain.
 
+#### Snapshot statistics
+
+Snapshot statistics files are valid [Puffin files](../puffin-spec). Statistics are informational. A reader can choose to
+ignore statistics information. Statistics support is not required to read the table correctly.
+
+Statistics files metadata within `snapshot-statistics` table metadata field is a struct with the following fields:
```

Comment:
I don't think that these should be "snapshot statistics" since these aren't part of `snapshot`. These are actually table statistics that are associated with a particular snapshot of the table. I think that `statistics` or `table-statistics` are better names.